### DISCUSSION TITLE
Uninstall Wordpress in Laragon

### DISCUSSION THREAD

**Discussion by Anthony1070** [leokhoa/laragon#483] [Q&A]
Hi,

I have installed Wordpress in Laragon using the shortcut Quick App > Wordpress

If I want disinstall it completely, how can I do? (right now I'm uninstalling and reinstalling Laragon each time).

Many thanks in advance!

A.

**Comment by dustinatx** [comment]
Menu > Tools > Delete project > [the WordPress project]

**Comment by Anthony1070** [comment]
Thanks a lot Dustin!

Have a nice day.

A.
